tests/test_base_encoding.py

```diff
@@ -0,0 +1,435 @@
+    ("Hallo, hoe gaat het met u?", [79178, 11, 46976, 69145, 9194, 2322, 577, 30]),
+    ("Hei, hvordan har du det?", [1548, 72, 11, 63259, 4960, 3930, 3474, 30]),
+    ("Tere, kuidas sul läheb?", [51, 486, 11, 597, 2480, 300, 26858, 31105, 383, 65, 30]),
+    ("Привет, как дела?", [54745, 28089, 8341, 11, 52770, 95369, 1506, 30]),
+    ("Szia, hogy vagy?", [50, 68151, 11, 33506, 62632, 30]),
+    ("Moien, wéi geet et Iech?", [26694, 3675, 11, 289, 978, 72, 3980, 295, 1880, 358, 4842, 30]),
+    ("Salve, come va?", [17691, 588, 11, 2586, 11412, 30]),
+    ("Labdien, kā jums klājas?", [30146, 67, 3675, 11, 597, 31757, 503, 6370, 20839, 31757, 30826, 30]),
+    ("Sveikas, kaip sekasi?", [50, 588, 1609, 300, 11, 16909, 575, 44934, 10426, 30]),
+    ("Sveiki, kā jums klājas?", [50, 588, 7723, 11, 597, 31757, 503, 6370, 20839, 31757, 30826, 30]),
+    ("Hallå, hur mår du?", [72945, 3870, 11, 13113, 296, 18382, 3930, 30]),
+    ("Здраво, како си?", [36551, 7094, 28086, 1482, 11, 52770, 1482, 5524, 1840, 30]),
+    ("czesc, jak sie masz?", [14088, 3380, 11, 19958, 10112, 9427, 89, 30]),
+    ("Halló, hvernig hefur þú það?", [72945, 1832, 11, 305, 76852, 343, 568, 84001, 80707, 6792, 80707, 64, 68800, 30]),
+    ("Salut, comment vas-tu?", [17691, 332, 11, 4068, 44496, 2442, 84, 30]),
+    ("Salam, nasılsın?", [50, 17243, 11, 17580, 3862, 4835, 16507, 30]),
+    ("Здравейте, как сте?", [36551, 7094, 28086, 21708, 51627, 11, 52770, 18868, 1532, 30]),
+    ("Dobrý den, jak se máte?", [35, 677, 81, 20195, 3453, 11, 19958, 513, 29830, 668, 30]),
+    ("Zdravo, kako ste?", [57, 3696, 28316, 11, 91617, 4179, 30]),
+    ("Здравствуйте, как поживаете?", [36551, 7094, 28086, 20812, 83680, 51627, 11, 52770, 5173, 21956, 28089, 28007, 1532, 30]),
+    ("Merhaba, nasılsın?", [27814, 10796, 64, 11, 17580, 3862, 4835, 16507, 30]),
+    ("Вітаю, як поживаєте?", [16604, 27385, 44613, 12182, 11, 46410, 4898, 5173, 21956, 28089, 1506, 141, 242, 51627, 30]),
+    ("Сәлеметсіз бе, сіз қалайсыз?", [19871, 143, 247, 37131, 8341, 2297, 27385, 9136, 14391, 1532, 11, 5524, 27385, 9136, 220, 142, 249, 16331, 19039, 57319, 9136, 30]),
+    ("Hallo, hou gaat het met jou?", [79178, 11, 305, 283, 69145, 9194, 2322, 28068, 30]),
+    ("Hallå, hur mår ni?", [72945, 3870, 11, 13113, 296, 18382, 13080, 30]),
+    ("Прывітанне, як пажываеце?", [54745, 35667, 27385, 1830, 7486, 79862, 11, 46410, 4898, 5173, 38657, 35667, 1506, 1532, 10589, 1532, 30]),
+    ("Olá, como vai você?", [43819, 1995, 11, 8112, 40586, 25738, 30]),
+    ("Здравствуй, как поживаете?", [36551, 7094, 28086, 20812, 83680, 11, 52770, 5173, 21956, 28089, 28007, 1532, 30]),
+    ("Hola, ¿cómo estás?", [69112, 11, 29386, 66, 72561, 1826, 7206, 30]),
+]
+# @formatter:on
+
+@pytest.mark.parametrize("input_text,expected_output", cl100k_test_data)
+def test_cl100k_base_prompts_encoding(input_text, expected_output):
```

Comment:
Parameterized tests to make the failure clear:
<img width="500" alt="image" src="https://github.com/openai/tiktoken/assets/1841944/94e01b4e-a861-42c2-acf6-3baf92060c40">